### PR TITLE
Story of JD developer was added

### PR DESCRIPTION
**Pull request by ajaxKpi**
Please uprove my story in Ukrainian, @casey-collab

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

File: ukrainian/JDstory/JDstory.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+�ЫІсторія просто�го JD розробника :)
```
Comment (casey-collab) [inline ukrainian/JDstory/JDstory.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline ukrainian/JDstory/JDstory.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
It looks like the file `ukrainian/JDstory/.swp` is empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to this file, or delete the option that leads to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**